### DISCUSSION TITLE
I want to change the display contents of nuxt-spa-loading.

### DISCUSSION THREAD

**Discussion by t-murotani** [nuxt/nuxt#22137] [Questions]
I am making SPA using NUXT3.

Each page I create is working as expected, but when I reload the page, I'm annoyed by the content showing up before rendering.

This happens even for <a> link transitions without <nuxt-link>.

Specifically, below is where the NUXT logo is displayed in SVG before rendering.

`<svg class="nuxt-spa-loading" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 37 25" fill="none" width="80"><path d="M24.236 22.006h10.742L25.563 5.822l-8.979 14.31a4 4 0 0 1-3.388 1.874H2.978l11.631-20 5.897 10.567"></path></svg>`

I want this to be a commonly used loading image or site logo. A blank page is fine.

Can someone teach me how to deal with this?

**Comment by arashsheyda** [comment]
@t-murotani Did I undrestand correctly, you want to change the Spa Loading template design?
if you want to change that you can create a file `~/app/spa-loading-template.html`

read more here: [spaloadingtemplate](https://nuxt.com/docs/api/configuration/nuxt-config#spaloadingtemplate)

**Comment by t-murotani** [comment]
@arashsheyda Thank you for the advice!

I've tried the overwriting method, but it doesn't work well because my overwriting method is bad.

```
$ find ./ -name "*spa-loading*"
./node_modules/@nuxt/ui-templates/dist/templates/spa-loading-icon.vue
./node_modules/@nuxt/ui-templates/dist/templates/spa-loading-icon.mjs
./node_modules/@nuxt/ui-templates/dist/templates/spa-loading-icon.d.ts

```

Since it is

```
spa-loading-template.html
spa-loading-template.vue
spa-loading-icon-template.html
spa-loading-icon.vue
```

I tried setting the file. . .

Modified nuxt.config.ts.

`spaLoadingTemplate: '~/app/spa-loading-template.html'`

**Comment by t-murotani** [comment]
Sorry..

> You can set this to false to disable any loading indicator.

I was not able to change the template content, but I was able to view the blank page.

thank you!
